docs: how changes in the spec are introduced (#488)
asyncapi/community: docs/README.md

> __docs__ folder for community related documentation. 

Reviewer: Sorry, I don't understand why this file is being added?

Author: To indicate that the docs folder basically holds all docs related to community

Reviewer: but why do are have a readme page with only a phrase written on it? that doesn't seem complete... lmk if that makes more sense now :)

Author: Yeah, it does makes sense :)

Author: Btw, I couldn't think of a long readME for the docs 😄

> # Community Documetation

Reviewer: ```suggestion
# Community Documentation
```

> The community documentation directory helps the community collaboratively maintain a collection of helpful community-related documentation, from how to start contributing to AsyncAPI to how it leverages other technologies to build and enhance its software.

Reviewer: ```suggestion
The community documentation directory helps the community collaboratively maintain a collection of helpful community-related documentation. From how to start contributing to AsyncAPI, becoming a TSC member, and more.
```

Some of those topics were **technical** docs topics, not **community** docs.

> The community documentation directory helps the community maintains collaboratively a collection of helpful community-related documentations from how to start contributing to AsyncAPI to how it leverages other technologies to build and enhance it's software.

Reviewer: ```suggestion
The community documentation directory helps the community collaboratively maintain a collection of helpful community-related documentation, from how to start contributing to AsyncAPI to how it leverages other technologies to build and enhance its software.
```
